### DISCUSSION TITLE
大佬 您有时间的话可以制作一个美漫的站点规则吗？

### DISCUSSION THREAD

**Discussion by zyx092213** [xiaohucode/yidaRule#33] [Ideas]
https://allporncomic.com/porncomic/ 就是这个站点，用您之前做的那个ios版的亦搜 这个站点的规则可能会爆内存，一加载大量图片就闪退

**Comment by nhgfdsaz123** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
